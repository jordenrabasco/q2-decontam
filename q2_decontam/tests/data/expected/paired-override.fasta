>GCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTCTGTCGCGTCGGCTGTGAAAACTCGGGGCTCAACTCCGAGCTTGCAGCCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGG
GCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTCTGTCGCGTCGGCTGTGAAAACTCGGGGCTCAACTCCGAGCTTGCAGCCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGG
>GCGAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCGTGTAGGCGGTCTGGTAAGTCTGCCGTGAAAACCCGGGGCTCAACCCCGGGCGTGCGGTGGATACTGCCAGGCTAGAGGGTGGTAGAGGCGAGTGGAATTCCCGGTGTAGCGGTGAAATGCGCAGATATCGGGAGGAACACCAGTAGCGAAGGCGGCTCGCTGGGCCATTCCTGACGCTGAGACGCGAAAGCTAGGGG
GCGAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCGTGTAGGCGGTCTGGTAAGTCTGCCGTGAAAACCCGGGGCTCAACCCCGGGCGTGCGGTGGATACTGCCAGGCTAGAGGGTGGTAGAGGCGAGTGGAATTCCCGGTGTAGCGGTGAAATGCGCAGATATCGGGAGGAACACCAGTAGCGAAGGCGGCTCGCTGGGCCATTCCTGACGCTGAGACGCGAAAGCTAGGGG
>GCGAGCGTTAATCGGAATTACTGGGCGTAAAGGGCGCGTAGGTGGTTGGGTAAGTCGGGTGTGAAAGCCCTGGGCTTAACCTGGGAATGGCATTCGAGACTACCTATCTAGAGTCTGGTAGAGGGAAGTGGAATTTCCGGTGTAGCGGTGAAATGTGTAGATATCGGAAGGAACACCAGTGGCGAAGGCGACTTCCTGGACCAAGACTGACGCTGAGGCGCGAAAGCGTGGGG
GCGAGCGTTAATCGGAATTACTGGGCGTAAAGGGCGCGTAGGTGGTTGGGTAAGTCGGGTGTGAAAGCCCTGGGCTTAACCTGGGAATGGCATTCGAGACTACCTATCTAGAGTCTGGTAGAGGGAAGTGGAATTTCCGGTGTAGCGGTGAAATGTGTAGATATCGGAAGGAACACCAGTGGCGAAGGCGACTTCCTGGACCAAGACTGACGCTGAGGCGCGAAAGCGTGGGG
>GCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTCTGTTACGTCGGCTGTGAAAACTCGGGGCTCAACCCTGAGCCTGCAGTCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGG
GCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTCTGTTACGTCGGCTGTGAAAACTCGGGGCTCAACCCTGAGCCTGCAGTCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGG
>CCGAGCGTTGTCCGGAATCATTGGGCGTAAAGAGCGCGTAGGCGGCTATCCACGTCGGGTGTGAAAGCTCAGAGCTCAACTCTGAGAGGCCATTCGATACGGGATAGCTAGAGGCAGCTAGGGGAGAGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGCTCTCTGGGGCTGATCTGACGCTGAGGCGCGAAAGCTGGGGG
CCGAGCGTTGTCCGGAATCATTGGGCGTAAAGAGCGCGTAGGCGGCTATCCACGTCGGGTGTGAAAGCTCAGAGCTCAACTCTGAGAGGCCATTCGATACGGGATAGCTAGAGGCAGCTAGGGGAGAGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGCTCTCTGGGGCTGATCTGACGCTGAGGCGCGAAAGCTGGGGG
>GCAAGCGTTGTCCGGATTCATTGGGCGTAAAGAGCTCGTAGGCGGCTTGGCAAGTCGGGTGTGAAAACTTCAGGCTCAACCTGGAGATGCCACCCGAGACTGCTATGGCTAGAGTCCGGTAGGGGACCACGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTAGCGAAGGCGGTGGTCTGGGCCGGCACTGACGCTGAGGAGCGAAAGCGTGGGT
GCAAGCGTTGTCCGGATTCATTGGGCGTAAAGAGCTCGTAGGCGGCTTGGCAAGTCGGGTGTGAAAACTTCAGGCTCAACCTGGAGATGCCACCCGAGACTGCTATGGCTAGAGTCCGGTAGGGGACCACGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTAGCGAAGGCGGTGGTCTGGGCCGGCACTGACGCTGAGGAGCGAAAGCGTGGGT
>GCAAGCGTTGTCCGGAATCATTGGGCGTAAAGAGCGCGTAGGCGGTCCGGTAAGTCCATCGTGAAAGCCAAGGGCTCAACCCTTGGATGCCGGTGGATACTGTCGGGCTAGAGTCCGGAAGAGGCGAGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAAGAACACCTATGGCGAAGGCAGCTCGCTGGGACGGAACTGACGCTGAGGCGCGAAAGCGTGGGG
GCAAGCGTTGTCCGGAATCATTGGGCGTAAAGAGCGCGTAGGCGGTCCGGTAAGTCCATCGTGAAAGCCAAGGGCTCAACCCTTGGATGCCGGTGGATACTGTCGGGCTAGAGTCCGGAAGAGGCGAGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAAGAACACCTATGGCGAAGGCAGCTCGCTGGGACGGAACTGACGCTGAGGCGCGAAAGCGTGGGG
>GCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTTTGTCGCGTCGGCTGTGAAAACTCGGGGCTCAACTCTGAGCTTGCAGTCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGG
GCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTTTGTCGCGTCGGCTGTGAAAACTCGGGGCTCAACTCTGAGCTTGCAGTCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGG
>GCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTTTGTCGCGTCGGCTGTGAAAACTCGGGGCTCAACTCCGAGCTTGCAGTCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGAAACTGACGCTGAGGAGCGAAAGCGTGGGG
GCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTTTGTCGCGTCGGCTGTGAAAACTCGGGGCTCAACTCCGAGCTTGCAGTCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGAAACTGACGCTGAGGAGCGAAAGCGTGGGG
>GCGAGCGTTGTCCGGATTTATTGGGCGTAAAGAGCTCGTAGGCGGCCGTTCGCGTCGGATGTGAAAGCTCAAGGCTCAACCTTGAGAGGCCATTCGATACGGGACGGCTGGAGGCAGGCAGGGGAGAGTGGAATTCCTGGTGTAGCGGTGGAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGCTCTCTGGGCCTGTTCTGACGCTGAGGAGCGAAAGCTGGGGT
GCGAGCGTTGTCCGGATTTATTGGGCGTAAAGAGCTCGTAGGCGGCCGTTCGCGTCGGATGTGAAAGCTCAAGGCTCAACCTTGAGAGGCCATTCGATACGGGACGGCTGGAGGCAGGCAGGGGAGAGTGGAATTCCTGGTGTAGCGGTGGAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGCTCTCTGGGCCTGTTCTGACGCTGAGGAGCGAAAGCTGGGGT
>CCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTCCGTTAAGTCGGAAGTGAAATCTCAAGGCTCAACCTTGAAATTGCTTTCGATACTGGCGGACTTGAGGCAGGTAGGGGAAGGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCAGTGGCGAAGGCGGCCTTCTGGGCCTGTACTGACGCTGAGGAGCGAAAGCGTGGGG
CCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTCCGTTAAGTCGGAAGTGAAATCTCAAGGCTCAACCTTGAAATTGCTTTCGATACTGGCGGACTTGAGGCAGGTAGGGGAAGGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCAGTGGCGAAGGCGGCCTTCTGGGCCTGTACTGACGCTGAGGAGCGAAAGCGTGGGG
>GCAAGCGTTATCCGGATTTATTGGGTTTAAAGGGTGCGTAGGCGGCCTGTTAAGTCAGGGGTGAAAGGTTGCGGCTTAACCGCAGAATTGCCTTTGATACTGGCGGGCTTGAATAAAGTTGAGGTAGGCGGAATGTGGCAAGTAGCGGTGAAATGCATAGAGATGCCACAGAACACCGATTGCGAAGGCAGCTTACTAAGCTTTGATTGACGCTGAGGCACGAAAGCGTGGGG
GCAAGCGTTATCCGGATTTATTGGGTTTAAAGGGTGCGTAGGCGGCCTGTTAAGTCAGGGGTGAAAGGTTGCGGCTTAACCGCAGAATTGCCTTTGATACTGGCGGGCTTGAATAAAGTTGAGGTAGGCGGAATGTGGCAAGTAGCGGTGAAATGCATAGAGATGCCACAGAACACCGATTGCGAAGGCAGCTTACTAAGCTTTGATTGACGCTGAGGCACGAAAGCGTGGGG
>CCAAGCGTTGTCCGGATTTATTGGGCGTAAAGAGCTCGTAGGCGGCTGTTCGCGTCGGATGTGAAAGCTCAGAGCTCAACTCTGAGAGGCCATTCGATACGGGATAGCTAGAGGTAGGTAGGGGAGATCGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGATCTCTGGGCCTTACCTGACGCTGAGGAGCGAAAGCTGGGGG
CCAAGCGTTGTCCGGATTTATTGGGCGTAAAGAGCTCGTAGGCGGCTGTTCGCGTCGGATGTGAAAGCTCAGAGCTCAACTCTGAGAGGCCATTCGATACGGGATAGCTAGAGGTAGGTAGGGGAGATCGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGATCTCTGGGCCTTACCTGACGCTGAGGAGCGAAAGCTGGGGG
>GCGAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCGTGTAGGCGGCCTGGTAAGTCTGCTGTGAAACCTTGGGGCTCAACCCCAAGCGTGCAGTGGATACTGCTAGGCTAGAGGATGGTAGAGGCGAGTGGAATTCCCGGTGTAGCGGTGAAATGCGCAGATATCGGGAGGAACACCAGTAGCGAAGGCGGCTCGCTGGGCCATTCCTGACGCTGAGACGCGAAAGCTAGGGG
GCGAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCGTGTAGGCGGCCTGGTAAGTCTGCTGTGAAACCTTGGGGCTCAACCCCAAGCGTGCAGTGGATACTGCTAGGCTAGAGGATGGTAGAGGCGAGTGGAATTCCCGGTGTAGCGGTGAAATGCGCAGATATCGGGAGGAACACCAGTAGCGAAGGCGGCTCGCTGGGCCATTCCTGACGCTGAGACGCGAAAGCTAGGGG
>GCAAGCGTTGTCCGGATTTATTGGGCGTAAAGAGCTCGTAGGCGGCCTGGTGAGTCGGGTGTGAAAGCCCGAGGCTCAACCTCGGAATTGCATTCGATACTGCTGGGCTTGAGGCAGGTAGGGGAGGATGGAATTCCCGGTGTAGCGGTGGAATGCGCAGATATCGGGAGGAACACCTGCGGCGAAGGCGGTCCTCTGGGCCTGTCCTGACGCTGAGGAGCGAAAGCGTGGGG
GCAAGCGTTGTCCGGATTTATTGGGCGTAAAGAGCTCGTAGGCGGCCTGGTGAGTCGGGTGTGAAAGCCCGAGGCTCAACCTCGGAATTGCATTCGATACTGCTGGGCTTGAGGCAGGTAGGGGAGGATGGAATTCCCGGTGTAGCGGTGGAATGCGCAGATATCGGGAGGAACACCTGCGGCGAAGGCGGTCCTCTGGGCCTGTCCTGACGCTGAGGAGCGAAAGCGTGGGG
>GCGAGCGTTAATCGGAATTACTGGGCGTAAAGGGCGCGTAGGCGGTTGGGTAAGTCGGATGTGAAAGCCCTGGGCTTAACCTGGGAATGGCATTCGAGACTACCTATCTAGAGTCTGGTAGAGGGAAGTGGAATTTCCGGTGTAGCGGTGAAATGTGTAGATATCGGAAGGAACACCAGTGGCGAAGGCGACTTCCTGGACCAAGACTGACGCTGAGGCGCGAAAGCGTGGGG
GCGAGCGTTAATCGGAATTACTGGGCGTAAAGGGCGCGTAGGCGGTTGGGTAAGTCGGATGTGAAAGCCCTGGGCTTAACCTGGGAATGGCATTCGAGACTACCTATCTAGAGTCTGGTAGAGGGAAGTGGAATTTCCGGTGTAGCGGTGAAATGTGTAGATATCGGAAGGAACACCAGTGGCGAAGGCGACTTCCTGGACCAAGACTGACGCTGAGGCGCGAAAGCGTGGGG
>GCGAGCGTTGTCCGGAATCACTGGGCGTAAAGGGCGCGTAGGCGGCCTGATAAGTAGGGGGTGAAATCCTGCGGCTTAACCGCAGGGCTGCCTTCTAAACTGTCAGGCTCGAGCACAGTAGAGGCAGGTGGAATTCCCGGTGTAGCGGTGGAATGCGTAGAGATCGGGAAGAACATCAGTGGCGAAGGCGGCCTGCTGGGCTGTTGCTGACGCTGAGGCGCGACAGCGTGGGG
GCGAGCGTTGTCCGGAATCACTGGGCGTAAAGGGCGCGTAGGCGGCCTGATAAGTAGGGGGTGAAATCCTGCGGCTTAACCGCAGGGCTGCCTTCTAAACTGTCAGGCTCGAGCACAGTAGAGGCAGGTGGAATTCCCGGTGTAGCGGTGGAATGCGTAGAGATCGGGAAGAACATCAGTGGCGAAGGCGGCCTGCTGGGCTGTTGCTGACGCTGAGGCGCGACAGCGTGGGG
>GCGAGCGTTAATCGGAATTACTGGGCGTAAAGCGCGCGTAGGCGGCTTTGCAAGTCGGGTGTGAAATCCCCAGGCTTAACCTGGGAACTGCATTCGAGACTGCATTGCTAGAGTATGGGAGAGGGAAGTGGAATTTCCGGTGTAGCGGTGAAATGCGTAGATATCGGAAGGAACATCAGTGGCGAAAGCGACTTCCTGGACCAATACTGACGCTCATGTGCGAAAGCGTGGGG
GCGAGCGTTAATCGGAATTACTGGGCGTAAAGCGCGCGTAGGCGGCTTTGCAAGTCGGGTGTGAAATCCCCAGGCTTAACCTGGGAACTGCATTCGAGACTGCATTGCTAGAGTATGGGAGAGGGAAGTGGAATTTCCGGTGTAGCGGTGAAATGCGTAGATATCGGAAGGAACATCAGTGGCGAAAGCGACTTCCTGGACCAATACTGACGCTCATGTGCGAAAGCGTGGGG
