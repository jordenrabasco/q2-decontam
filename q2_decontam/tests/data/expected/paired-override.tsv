# Constructed from biom file
#OTU ID	BAQ2420.2	BAQ3473.2	BAQ3473.3	BAQ4166.2	BAQ4166.3	PAP130.2	YUN1005.3	YUN1242.1	YUN1242.3	YUN3153.3
GCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTCTGTCGCGTCGGCTGTGAAAACTCGGGGCTCAACTCCGAGCTTGCAGCCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGG	0.0	0.0	0.0	0.0	0.0	0.0	0.0	31.0	0.0	0.0
GCGAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCGTGTAGGCGGTCTGGTAAGTCTGCCGTGAAAACCCGGGGCTCAACCCCGGGCGTGCGGTGGATACTGCCAGGCTAGAGGGTGGTAGAGGCGAGTGGAATTCCCGGTGTAGCGGTGAAATGCGCAGATATCGGGAGGAACACCAGTAGCGAAGGCGGCTCGCTGGGCCATTCCTGACGCTGAGACGCGAAAGCTAGGGG	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	26.0	0.0
GCGAGCGTTAATCGGAATTACTGGGCGTAAAGGGCGCGTAGGTGGTTGGGTAAGTCGGGTGTGAAAGCCCTGGGCTTAACCTGGGAATGGCATTCGAGACTACCTATCTAGAGTCTGGTAGAGGGAAGTGGAATTTCCGGTGTAGCGGTGAAATGTGTAGATATCGGAAGGAACACCAGTGGCGAAGGCGACTTCCTGGACCAAGACTGACGCTGAGGCGCGAAAGCGTGGGG	21.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
GCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTCTGTTACGTCGGCTGTGAAAACTCGGGGCTCAACCCTGAGCCTGCAGTCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGG	0.0	0.0	0.0	0.0	0.0	0.0	21.0	0.0	0.0	0.0
CCGAGCGTTGTCCGGAATCATTGGGCGTAAAGAGCGCGTAGGCGGCTATCCACGTCGGGTGTGAAAGCTCAGAGCTCAACTCTGAGAGGCCATTCGATACGGGATAGCTAGAGGCAGCTAGGGGAGAGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGCTCTCTGGGGCTGATCTGACGCTGAGGCGCGAAAGCTGGGGG	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	20.0	0.0
GCAAGCGTTGTCCGGATTCATTGGGCGTAAAGAGCTCGTAGGCGGCTTGGCAAGTCGGGTGTGAAAACTTCAGGCTCAACCTGGAGATGCCACCCGAGACTGCTATGGCTAGAGTCCGGTAGGGGACCACGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTAGCGAAGGCGGTGGTCTGGGCCGGCACTGACGCTGAGGAGCGAAAGCGTGGGT	0.0	0.0	0.0	0.0	0.0	0.0	17.0	0.0	0.0	0.0
GCAAGCGTTGTCCGGAATCATTGGGCGTAAAGAGCGCGTAGGCGGTCCGGTAAGTCCATCGTGAAAGCCAAGGGCTCAACCCTTGGATGCCGGTGGATACTGTCGGGCTAGAGTCCGGAAGAGGCGAGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAAGAACACCTATGGCGAAGGCAGCTCGCTGGGACGGAACTGACGCTGAGGCGCGAAAGCGTGGGG	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	17.0
GCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTTTGTCGCGTCGGCTGTGAAAACTCGGGGCTCAACTCTGAGCTTGCAGTCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGATACTGACGCTGAGGAGCGAAAGCGTGGGG	0.0	0.0	0.0	0.0	0.0	0.0	15.0	0.0	0.0	0.0
GCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTTTGTCGCGTCGGCTGTGAAAACTCGGGGCTCAACTCCGAGCTTGCAGTCGATACGGGCAGACTAGAGTTCGGCAGGGGAGACTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGGTCTCTGGGCCGAAACTGACGCTGAGGAGCGAAAGCGTGGGG	0.0	0.0	0.0	0.0	0.0	0.0	0.0	13.0	0.0	0.0
GCGAGCGTTGTCCGGATTTATTGGGCGTAAAGAGCTCGTAGGCGGCCGTTCGCGTCGGATGTGAAAGCTCAAGGCTCAACCTTGAGAGGCCATTCGATACGGGACGGCTGGAGGCAGGCAGGGGAGAGTGGAATTCCTGGTGTAGCGGTGGAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGCTCTCTGGGCCTGTTCTGACGCTGAGGAGCGAAAGCTGGGGT	0.0	0.0	0.0	0.0	0.0	0.0	12.0	0.0	0.0	0.0
CCAAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCTCGTAGGCGGTCCGTTAAGTCGGAAGTGAAATCTCAAGGCTCAACCTTGAAATTGCTTTCGATACTGGCGGACTTGAGGCAGGTAGGGGAAGGTGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCAGTGGCGAAGGCGGCCTTCTGGGCCTGTACTGACGCTGAGGAGCGAAAGCGTGGGG	0.0	0.0	0.0	0.0	0.0	0.0	0.0	12.0	0.0	0.0
GCAAGCGTTATCCGGATTTATTGGGTTTAAAGGGTGCGTAGGCGGCCTGTTAAGTCAGGGGTGAAAGGTTGCGGCTTAACCGCAGAATTGCCTTTGATACTGGCGGGCTTGAATAAAGTTGAGGTAGGCGGAATGTGGCAAGTAGCGGTGAAATGCATAGAGATGCCACAGAACACCGATTGCGAAGGCAGCTTACTAAGCTTTGATTGACGCTGAGGCACGAAAGCGTGGGG	0.0	0.0	11.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
CCAAGCGTTGTCCGGATTTATTGGGCGTAAAGAGCTCGTAGGCGGCTGTTCGCGTCGGATGTGAAAGCTCAGAGCTCAACTCTGAGAGGCCATTCGATACGGGATAGCTAGAGGTAGGTAGGGGAGATCGGAATTCCTGGTGTAGCGGTGAAATGCGCAGATATCAGGAGGAACACCGGTGGCGAAGGCGGATCTCTGGGCCTTACCTGACGCTGAGGAGCGAAAGCTGGGGG	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	11.0
GCGAGCGTTGTCCGGAATTATTGGGCGTAAAGAGCGTGTAGGCGGCCTGGTAAGTCTGCTGTGAAACCTTGGGGCTCAACCCCAAGCGTGCAGTGGATACTGCTAGGCTAGAGGATGGTAGAGGCGAGTGGAATTCCCGGTGTAGCGGTGAAATGCGCAGATATCGGGAGGAACACCAGTAGCGAAGGCGGCTCGCTGGGCCATTCCTGACGCTGAGACGCGAAAGCTAGGGG	0.0	0.0	0.0	0.0	0.0	0.0	0.0	9.0	0.0	0.0
GCAAGCGTTGTCCGGATTTATTGGGCGTAAAGAGCTCGTAGGCGGCCTGGTGAGTCGGGTGTGAAAGCCCGAGGCTCAACCTCGGAATTGCATTCGATACTGCTGGGCTTGAGGCAGGTAGGGGAGGATGGAATTCCCGGTGTAGCGGTGGAATGCGCAGATATCGGGAGGAACACCTGCGGCGAAGGCGGTCCTCTGGGCCTGTCCTGACGCTGAGGAGCGAAAGCGTGGGG	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	7.0	0.0
GCGAGCGTTAATCGGAATTACTGGGCGTAAAGGGCGCGTAGGCGGTTGGGTAAGTCGGATGTGAAAGCCCTGGGCTTAACCTGGGAATGGCATTCGAGACTACCTATCTAGAGTCTGGTAGAGGGAAGTGGAATTTCCGGTGTAGCGGTGAAATGTGTAGATATCGGAAGGAACACCAGTGGCGAAGGCGACTTCCTGGACCAAGACTGACGCTGAGGCGCGAAAGCGTGGGG	0.0	0.0	6.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0
GCGAGCGTTGTCCGGAATCACTGGGCGTAAAGGGCGCGTAGGCGGCCTGATAAGTAGGGGGTGAAATCCTGCGGCTTAACCGCAGGGCTGCCTTCTAAACTGTCAGGCTCGAGCACAGTAGAGGCAGGTGGAATTCCCGGTGTAGCGGTGGAATGCGTAGAGATCGGGAAGAACATCAGTGGCGAAGGCGGCCTGCTGGGCTGTTGCTGACGCTGAGGCGCGACAGCGTGGGG	0.0	0.0	0.0	0.0	6.0	0.0	0.0	0.0	0.0	0.0
GCGAGCGTTAATCGGAATTACTGGGCGTAAAGCGCGCGTAGGCGGCTTTGCAAGTCGGGTGTGAAATCCCCAGGCTTAACCTGGGAACTGCATTCGAGACTGCATTGCTAGAGTATGGGAGAGGGAAGTGGAATTTCCGGTGTAGCGGTGAAATGCGTAGATATCGGAAGGAACATCAGTGGCGAAAGCGACTTCCTGGACCAATACTGACGCTCATGTGCGAAAGCGTGGGG	0.0	0.0	0.0	0.0	3.0	0.0	0.0	0.0	0.0	0.0